api/handlers_elastigroups.go

```diff
@@ -79,7 +79,44 @@ func (s *server) ElastigroupShowHandler(w http.ResponseWriter, r *http.Request)
 	w.Write(j)
 }
 
-// ElastgroupCreateHandler handles creating an elastigroup in SpotInst
+// ElastigroupUpdateHandler handles updating an elastigroup in SpotInst
+func (s *server) ElastigroupUpdateHandler(w http.ResponseWriter, r *http.Request) {
+	w = LogWriter{w}
+	vars := mux.Vars(r)
+	account := vars["account"]
+	esService, ok := s.elastigroupServices[account]
+	if !ok {
+		log.Errorf("account not found: %s", account)
+		return
+	}
+
+	req := aws.Group{}
+	err := json.NewDecoder(r.Body).Decode(&req)
+	if err != nil {
+		msg := fmt.Sprintf("cannot decode body into update elastigroup input: %s", err)
+		handleError(w, apierror.New(apierror.ErrBadRequest, msg, err))
+		return
+	}
+
+	output, err := esService.UpdateAWSElastigroup(r.Context(), &req)
```

Comment:
we should probably override the group id in the body before we pass it here

Reply:
complete